src/lib/pull-request.coffee

```diff
@@ -25,6 +25,15 @@ pr = (token, branch, body, cb) ->
   If I do something wrong, [blame a human](https://github.com/cfpb/hubot-capital-framework/issues/new).
 
   ![kitten gif](http://thecatapi.com/api/images/get?format=src&type=gif)
+
+  ## You are not
```

Comment:
Not what? Not done, I'm guessing?

Reply:
The previous heading says "I am a bot". Was just sticking to the playful nature of the bot interactions but I'm open to suggestions.

Reply:
Oh, hahah. Didn't realize that. Works for me, then.